packages/notebook/src/celllist.ts

```diff
@@ -182,8 +144,8 @@ class CellList implements IObservableUndoableVector<ICellModel> {
    * #### Undefined Behavior
    * An `index` which is non-integral or out of range.
    */
-  at(index: number): ICellModel {
-    return this._cellMap.get(this._cellOrder.at(index)) as ICellModel;
+  get(index: number): ICellModel {
+    return this._cellMap.get(this._cellOrder.get(index)) as ICellModel;
```

Comment:
Will we need to revisit this logic to make sure the two data structures are synchronized in this call sequence?